repose-aggregator/components/filters/keystone-v2/src/main/resources/META-INF/schema/config/keystone-v2.xsd

```diff
@@ -108,6 +108,13 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="set-rackers-groups" type="xs:string" use="optional">
```

Comment:
Also, there's still the question about what we want to do with regards to translating roles to groups. If we were to do that translation, there may not be a need for a configuration group value here.